src/views/Feedbacks.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <v-container fluid class="pa-0 mx-7">
+    <h1 class="my-5">Feedbacks</h1>
+    <v-row class="my-5"> <SlideOrizontalGroup /> </v-row>
+  </v-container>
+</template>
+
+<script>
+import SlideOrizontalGroup from "@/components/feedbacks/SlideOrizontalGroup";
+export default {
+  name: "Feedbacks",
+  components: { SlideOrizontalGroup },
+  data: () => ({}),
+  methods: {
+    goHome() {
```

Comment:
Questo metodo dove viene usato? A me non sembra di averlo visto, si può togliere?

Reply:
è in previsione della barra di ricerca, al prossimo commit dovrebbe servire